vscode/src/registry.json

```diff
@@ -0,0 +1,64 @@
+{
+  "knownPackages": [
+    {
+      "name": "Signed",
+      "description": "Defines types and functions to work with signed qubit-based integers.",
+      "dependency": {
+        "github": {
+          "owner": "microsoft",
+          "repo": "qsharp",
+          "refs": [
+            { "ref": "546b053", "notes": "latest stable" },
+            { "ref": "main", "notes": "nightly, unstable" }
+          ],
+          "path": "library/signed"
+        }
+      }
+    },
+    {
+      "name": "FixedPoint",
+      "description": "Types and functions for fixed-point arithmetic with qubits.",
+      "dependency": {
+        "github": {
+          "owner": "microsoft",
+          "repo": "qsharp",
+          "refs": [
+            { "ref": "546b053", "notes": "latest stable" },
```

Comment:
We should bump this after #2121 going in, as the library doesn't work without it.